prismic-model/js/teams.js

```diff
@@ -0,0 +1,14 @@
+import structuredText from './parts/structured-text';
+import text from './parts/text';
+
+const Teams = {
+  Contact: {
+    title: structuredText('Title', 'single', ['heading1']),
+    subtitle: text('Subtitle'),
```

Comment:
We're never going to want to put bits of markup in here?
Seems odd as the title allows it.

Reply:
Hmmmm… maybe I suppose. I came down on the side of it being YAGNI, but maybe that was too strong.